iroh-relay/src/main.rs

```diff
@@ -676,4 +735,28 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_access_config() -> TestResult {
+        let config = "
+            access = \"everyone\"
+        ";
+        let config = Config::from_str(config)?;
+        assert_eq!(config.access, AccessConfig::Everyone);
+
+        let mut rng = ChaCha8Rng::seed_from_u64(0);
+        let node_id = SecretKey::generate(&mut rng).public();
+
+        let config = format!(
+            "
+            access.allowlist = [
+              \"{node_id}\",
+            ]
```

Comment:
This kind of points out this is a rather whaky format.  So I tried:
```toml
access = "everyone"
access.allowlist = [
   "{node_id}",
]
```

And well, this happily fails.  Which is something at least I guess.

I'm a bit torn.  But I guess this is fine to use TOML syntax like this.  So if you like this I'm not objecting.

Reply:
it is what I expected 😅